microprofile.ls/com.redhat.microprofile.ls/src/main/resources/com/redhat/quarkus/snippets/quarkus-properties.json

```diff
@@ -25,5 +26,15 @@
 		"context": {
 			"dependency": "quarkus-jaeger"
 		}
+	},
+	"Add Hibernate ORM datasource properties": {
+		"prefix": "qhormd",
+		"body": [
+			"quarkus.hibernate-orm.database.generation=${5|none,drop-and-create,create,drop,update|}"
```

Comment:
I don't think having a 1 line snippet is worth it, when simply autocompleting quarkus.hibernate-orm.database.generation provides the same choices (I haven't checked but I assume it's the current behavior)

Reply:
I checked, and completing `quarkus.hibernate-orm.database.generation` doesn't give the list of options to cycle through like the snippet does. The available options are listed in the completion box though:
![MicroprofileIssue316Demo1](https://user-images.githubusercontent.com/22376627/82671456-e2648300-9c0c-11ea-82c8-70985684d96e.png)
I agree that a one line snippet doesn't make much sense, so I'll remove it.

Reply:
Indeed, `quarkus.hiberbate-orm.database.generation` has no value completion options for ow because it is defined as a String, and not an enum. But yes, I agree with the fact that having a one line snippet might not be worth it.

Maybe we can have a hibernate-specific snippet which also contains the datasource properties: 
```
# datasource configuration
quarkus.datasource.db-kind = postgresql
quarkus.datasource.username = hibernate
quarkus.datasource.password = hibernate
quarkus.datasource.jdbc.url = jdbc:postgresql://localhost:5432/hibernate_db

# drop and create the database at startup (use `update` to only update the schema)
quarkus.hibernate-orm.database.generation=drop-and-create
```
which is from the Quarkus Hibernate ORM guide: https://quarkus.io/guides/hibernate-orm

This would be fine because it seems that Hibernate ORM has quarkus-agroal as a dependency: https://github.com/quarkusio/quarkus/blob/2d8867d9f1481b02a94a334da86a2068cd4733a7/extensions/hibernate-orm/runtime/pom.xml#L20-L23, meaning that the datasource properties won't be unknown properties.

Or alternatively, we can add the hibernate-specific snippet after https://github.com/redhat-developer/quarkus-ls/issues/312 is fixed

Reply:
Eventually using the method from #312 sounds like a good idea. For the mean time, I could add another snippet with the same prefix that appears higher on the list when quarkus-hibernate-orm is loaded.

Reply:
I removed the hibernate orm snippet entirely